ac-source-css.el

```diff
@@ -0,0 +1,387 @@
+;; -*- mode: emacs-lisp; lexical-binding: t -*-
```

Comment:
You might want to add the header as well. Make sure you follow the multiple files package convention. See [here](https://www.emacswiki.org/emacs/MakingPackages).

Reply:
Of course. Thanks, I totally forgot about it. I'll have some time to work on this next weekend probably - will fix this and rename the file as you suggested then.